python/tk_multi_publish2/dialog.py

```diff
@@ -1195,17 +1368,47 @@ def _on_item_context_change(self, context):
         # are on context change lockdown.
         sync_required = False
 
+        # TODO: see todo below...
```

Comment:
Leaving this commented out and marked as TODO as per Rob. There are some things to consider with respect to changing this behavior that go beyond the scope of this initial api release. I'll do a full write up of this later this week.